tests/integration/scheduler/test_eval.py

```diff
@@ -753,15 +769,19 @@ def test_eval_minutes(self):
         ret = self.schedule.job_status(job_name)
         self.assertEqual(ret['_last_run'], run_time)
 
-        time.sleep(2)
+        # Give the job a chance to finish
+        while self.schedule.job_running(ret):
```

Comment:
Should this have some kind of timeout? What are the chances that something could goo wrong and it will run forever?

Reply:
Since the jobs are no longer running in the background this might not be necessary.  Will check and update.

Reply:
Looks like the previous failures on py2-centos-7 are passing with the change to not have the jobs run in the background and don't require the check to see if the job is still running.  This makes sense since it's running it right away and returning right away.